packages/features/bookings/lib/handleNewBooking.ts

```diff
@@ -2088,6 +2089,15 @@ async function handler(
   }
 
   if (!booking) throw new HttpError({ statusCode: 400, message: "Booking failed" });
+  if (!isDryRun) {
+    await createCalendarSyncTask({
+      results,
+      organizer: {
+        id: organizerUser.id,
+        organizationId: organizerOrganizationId ?? null,
+      },
+    });
```

Comment:
It has its own try-catch, so worst case we just log the error. No regression due to it.